### PR TITLE
Know Sure Thing indicator docs

### PR DESCRIPTION
**Pull request by quantum8**

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Topics/C/Community/Community-Workspaces/community-workspaces-013-api-data-fetcher-bot-demo-plugin-workspace.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
             "translations": [
                 {
                     "language": "RU",
-                    "text": "Разработчик",
+                    "text": "Разра��отчик",
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/C/Community/Community-Workspaces/community-workspaces-013-api-data-fetcher-bot-demo-plugin-workspace.json:29]: Does only github not know the letters marked with two question marks or superalgos too?

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/divergences-data-mine-013-know-sure-thing-oscillator.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
+    "topic": "Divergences Data Mine",
+    "pageNumber": 13,
+    "type": "Know Sure Thing Oscillator",
+    "definition": {
+        "text": "The Know Sure Thing Oscillator indicator was developed by Martin Pring in 1992 to make rate of change readings easier for traders to interpret.",
+        "updated": 1652685074675
+    },
+    "paragraphs": [
+        {
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/divergences-data-mine-013-know-sure-thing-oscillator.json:10]: Did this file happen on accident or is it planned to be a  new file?

**Review by gitforkgit — COMMENTED** [review]
Please read trough my two questions and answer if possible. 
Beside these two questions everything is ready to merge. :+1:

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/divergences-data-mine-013-know-sure-thing-oscillator.json (hunk 2 above)
Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/divergences-data-mine-013-know-sure-thing-oscillator.json:10]: This is a new file.

File: Projects/Foundations/Schemas/Docs-Topics/C/Community/Community-Workspaces/community-workspaces-013-api-data-fetcher-bot-demo-plugin-workspace.json (hunk 1 above)
Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/C/Community/Community-Workspaces/community-workspaces-013-api-data-fetcher-bot-demo-plugin-workspace.json:29]: That's really weird. I only edited the English in SA and used docs.save....so I don't know why the Russian translation was affected here.

Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/C/Community/Community-Workspaces/community-workspaces-013-api-data-fetcher-bot-demo-plugin-workspace.json]: Should be sorted now as i committed the change a couple hours ago. Not sure why it still shows here but it does have the outdated tag so I guess it's overwritten?

**Review by gitforkgit — APPROVED** [review]

**Comment by gitforkgit** [comment]
I have approved this Pull request. The @Superalgos/foundations-project needs to approve and then we can merge. :+1:

**Review by gitforkgit — APPROVED** [review]

**Pull request merged**